test/verify/check-machines

```diff
@@ -2825,7 +2825,7 @@ class TestMachines(MachineCase, StorageHelpers, NetworkHelpers):
         m.execute("virsh attach-interface --persistent VmNotInstalled bridge virbr0")
 
         # Change the os boot firmware configuration
-        supports_firmware_config = m.image in ['fedora-32', 'fedora-33', 'fedora-testing', 'rhel-8-3', 'rhel-8-3-distropkg', "rhel-8-4", 'debian-testing', 'ubuntu-stable', 'ubuntu-2004']
+        supports_firmware_config = m.image in ['fedora-32', 'fedora-33', 'fedora-testing', 'rhel-8-3', 'rhel-8-3-distropkg', 'debian-testing', 'ubuntu-stable', 'ubuntu-2004']
```

Comment:
This looks strange -- the effect is that rhel-8-4 would suddenly *stop* supporting firmware config. Also, by now we should certainly invert that list, I think debian-stable is the only remaining OS that does not support it. Such positive lists are prone to forgetting new ones.